src/viking.js

```diff
@@ -1,11 +1,125 @@
+// Link to pull: https://github.com/ironhack-labs/lab-javascript-vikings/pull/3889
+
+
 // Soldier
-class Soldier {}
+class Soldier {
+
+    constructor(health, strength){
+        this.health = health
+        this. strength = strength
+    }
+    attack(){
+        return this.strength
+
+    }
```

Comment:
I would add a line to separate the 2 methods
```suggestion
    }
    
```